unittests/ThunkFunctionalTests/CMakeLists.txt

```
@@ -0,0 +1,59 @@
+set(FUNCTIONAL_DEPENDS "")
+
+function(AddThunksTest Bin ThunksFile)
+  string(LENGTH "${ThunksFile}" ThunkFileLength)
+
+  set (ARGS
+    "-t" "${CMAKE_INSTALL_PREFIX}/lib/fex-emu/HostThunks"
+    "-j" "${CMAKE_INSTALL_PREFIX}/share/fex-emu/GuestThunks"
+    "-o" "stderr" "--no-silent" "-c" "irjit" "-n" "500"
+    )
+  if (ThunkFileLength EQUAL 0)
```

CMake is a really good scripting language (/s), so this works:
```suggestion
  if (NOT ThunkFile)
```

---

Oh right. Resolved.